plugin/src/main/java/org/opensearch/ml/plugin/MachineLearningPlugin.java

```diff
@@ -918,6 +921,9 @@ public List<Setting<?>> getSettings() {
                 MLCommonsSettings.ML_COMMONS_REMOTE_MODEL_ELIGIBLE_NODE_ROLES,
                 MLCommonsSettings.ML_COMMONS_LOCAL_MODEL_ELIGIBLE_NODE_ROLES,
                 MLCommonsSettings.ML_COMMONS_REMOTE_INFERENCE_ENABLED,
+                MLCommonsSettings.ML_COMMONS_REMOTE_INFERENCE_RETRY_ENABLED,
```

Comment:
Why we need to add a setting flag for this feature?

Reply:
I want to follow the best practice for contributing in ml-commons proposed by Yaliang
> For new features, suggest add flag setting to control enable/disable it. 

From my point, one of the use case for disable is to adjust the `max_connections` of the connectors

Reply:
We should consider if it's better to configure these in connectors, since different endpoint might need different strategies.

Reply:
Makes sense.

Reply:
Seems we can add to ConnectorClientConfig class

Reply:
Moved to ConnectorClientConfig in the latest commit.